### PR TITLE
atoms.dm and atoms_movable.dm var name upgrade

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
just some variable name upgrade

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -371,33 +371,33 @@
...
+	if(istype(current_turf.loc, /area/centcom))
 		return TRUE
 
 	//Check for centcom shuttles
-	for(var/A in SSshuttle.mobile)
-		var/obj/docking_port/mobile/M = A
-		if(M.launch_status == ENDGAME_LAUNCHED)
-			for(var/place in M.shuttle_areas)
+	for(var/moving_shuttle in SSshuttle.mobile)
+		var/obj/docking_port/mobile/mobile_docking_port = moving_shuttle
```
Comment (Kylerace) [inline code/game/atoms.dm:396]: as anything

File: code/game/atoms.dm [hunk 2]
```diff
@@ -371,33 +371,33 @@
...
+		if(mobile_docking_port.launch_status == ENDGAME_LAUNCHED)
+			for(var/place in mobile_docking_port.shuttle_areas)
```
Comment (Kylerace) [inline code/game/atoms.dm:398]: as anything

File: code/game/atoms.dm [hunk 3]
```diff
@@ -541,15 +542,15 @@
 
 	SEND_SIGNAL(source, COMSIG_REAGENTS_EXPOSE_ATOM, src, reagents, methods, volume_modifier, show_message)
 	for(var/reagent in reagents)
-		var/datum/reagent/R = reagent
-		. |= R.expose_atom(src, reagents[R])
+		var/datum/reagent/current_reagent = reagent
```
Comment (Kylerace) [inline code/game/atoms.dm:545]: as anything

File: code/game/atoms.dm [hunk 4]
```diff
@@ -641,24 +642,24 @@
 
 	if(custom_materials)
 		var/list/materials_list = list()
-		for(var/i in custom_materials)
-			var/datum/material/M = i
-			materials_list += "[M.name]"
+		for(var/material in custom_materials)
+			var/datum/material/current_material = material
```
Comment (Kylerace) [inline code/game/atoms.dm:646]: as anything

File: code/game/atoms.dm [hunk 5]
```diff
@@ -1029,18 +1030,18 @@
...
  *
  * Default behaviour is to loop through atom contents and call their HandleTurfChange() proc
  */
-/atom/proc/HandleTurfChange(turf/T)
-	for(var/a in src)
-		var/atom/A = a
-		A.HandleTurfChange(T)
+/atom/proc/HandleTurfChange(turf/changing_turf)
+	for(var/atom in src)
+		var/atom/current_atom = atom
```
Comment (Kylerace) [inline code/game/atoms.dm:1043]: as anything

File: code/game/atoms.dm [hunk 6]
```diff
@@ -1388,84 +1389,84 @@
...
 		if(processing_recipes.len)
-			process_recipes(user, I, processing_recipes)
-		if(QDELETED(I))
+			process_recipes(user, tool, processing_recipes)
+		if(QDELETED(tool))
 			return TRUE
 		switch(tool_type)
 			if(TOOL_CROWBAR)
-				act_result = crowbar_act(user, I,)
+				act_result = crowbar_act(user, tool,)
```
Comment (Kylerace) [inline code/game/atoms.dm:1406]:
```suggestion
				act_result = crowbar_act(user, tool)
```
what the fuck, why doesnt this error

File: code/game/atoms.dm [hunk 7]
```diff
@@ -1388,84 +1389,84 @@
...
+				act_result = analyzer_act(user, tool)
 	else
-		signal_result = SEND_SIGNAL(src, COMSIG_ATOM_SECONDARY_TOOL_ACT(tool_type), user, I)
+		signal_result = SEND_SIGNAL(src, COMSIG_ATOM_SECONDARY_TOOL_ACT(tool_type), user, tool)
 		if(signal_result & COMPONENT_BLOCK_TOOL_ATTACK) // The COMSIG_ATOM_TOOL_ACT signal is blocking the act
 			return TOOL_ACT_SIGNAL_BLOCKING
 		switch(tool_type)
 			if(TOOL_CROWBAR)
-				act_result = crowbar_act_secondary(user, I,)
+				act_result = crowbar_act_secondary(user, tool,)
```
Comment (Kylerace) [inline code/game/atoms.dm:1425]:
```suggestion
				act_result = crowbar_act_secondary(user, tool)
```

File: code/game/atoms.dm [hunk 8]
```diff
@@ -1388,84 +1389,84 @@
...
 ///Creates the radial and processes the selected option
-/atom/proc/ShowProcessingGui(mob/living/user, obj/item/I, list/possible_options)
+/atom/proc/ShowProcessingGui(mob/living/user, obj/item/processed_object, list/possible_options)
 	var/list/choices_to_options = list() //Dict of object name | dict of object processing settings
 	var/list/choices = list()
 
-	for(var/i in possible_options)
-		var/list/current_option = i
+	for(var/option in possible_options)
+		var/list/current_option = option
```
Comment (Kylerace) [inline code/game/atoms.dm:1456]: as anything

File: code/game/atoms.dm [hunk 9]
```diff
@@ -2063,8 +2064,8 @@
 	if (!source)
 		output += src
 	processed += src
-	for (var/o in orbiters?.orbiter_list)
-		var/atom/atom_orbiter = o
+	for (var/orbiter in orbiters?.orbiter_list)
+		var/atom/atom_orbiter = orbiter
```
Comment (Kylerace) [inline code/game/atoms.dm:2068]: as anything

File: code/game/atoms_movable.dm [hunk 10]
```diff
@@ -183,34 +183,34 @@
...
 
-/atom/movable/proc/onZImpact(turf/T, levels)
-	var/atom/highest = T
-	for(var/i in T.contents)
-		var/atom/A = i
-		if(!A.density)
+/atom/movable/proc/onZImpact(turf/impacted_turf, levels)
+	var/atom/highest = impacted_turf
+	for(var/current_atom in impacted_turf.contents)
+		var/atom/hurt_atom = current_atom
```
Comment (Kylerace) [inline code/game/atoms_movable.dm:189]: as anything

**Review by Kylerace — CHANGES_REQUESTED** [review]

File: code/game/atoms.dm [hunk 11]
```diff
@@ -641,24 +642,24 @@
...
+			materials_list += "[current_material.name]"
 		. += "<u>It is made out of [english_list(materials_list)]</u>."
 	if(reagents)
 		if(reagents.flags & TRANSPARENT)
 			. += "It contains:"
 			if(length(reagents.reagent_list))
 				if(user.can_see_reagents()) //Show each individual reagent
-					for(var/datum/reagent/R in reagents.reagent_list)
-						. += "[round(R.volume, 0.01)] units of [R.name]"
+					for(var/datum/reagent/current_reagent in reagents.reagent_list)
```
Comment (Watermelon914) [inline code/game/atoms.dm:654]: if these are 100% guaranteed to be reagents, put `as anything` for them

File: code/game/atoms.dm (hunk 4 above)
Comment (Watermelon914) [reply code/game/atoms.dm:646]: this was not properly resolved

File: code/game/atoms.dm (hunk 5 above)
Comment (Watermelon914) [reply code/game/atoms.dm:1043]: not properly resolved

File: code/game/atoms.dm (hunk 8 above)
Comment (Watermelon914) [reply code/game/atoms.dm:1456]: not properly addressed

File: code/game/atoms.dm [hunk 12]
```diff
@@ -1948,40 +1949,40 @@
...
+	if(turf_area.has_gravity) // Areas which always has gravity
+		return turf_area.has_gravity
 	else
 		// There's a gravity generator on our z level
-		if(GLOB.gravity_generators["[T.z]"])
+		if(GLOB.gravity_generators["[gravity_turf.z]"])
 			var/max_grav = 0
-			for(var/obj/machinery/gravity_generator/main/G in GLOB.gravity_generators["[T.z]"])
-				max_grav = max(G.setting,max_grav)
+			for(var/obj/machinery/gravity_generator/main/main_grav_gen in GLOB.gravity_generators["[gravity_turf.z]"])
```
Comment (Watermelon914) [inline code/game/atoms.dm:1982]: if this is guaranteed to be a gravity generator, do as anything

File: code/game/atoms.dm (hunk 9 above)
Comment (Watermelon914) [reply code/game/atoms.dm:2068]: not properly resolved

File: code/game/atoms_movable.dm (hunk 10 above)
Comment (Watermelon914) [reply code/game/atoms_movable.dm:189]: not properly resolved

File: code/game/atoms.dm (hunk 3 above)
Comment (Watermelon914) [reply code/game/atoms.dm:545]: not properly resolved

File: code/game/atoms.dm (hunk 1 above)
Comment (Watermelon914) [reply code/game/atoms.dm:396]: not properly resolved

File: code/game/atoms.dm [hunk 13]
```diff
@@ -641,24 +642,24 @@
...
+					for(var/datum/reagent/current_reagent in reagents.reagent_list)
+						. += "[round(current_reagent.volume, 0.01)] units of [current_reagent.name]"
 					if(reagents.is_reacting)
 						. += span_warning("It is currently reacting!")
 					. += span_notice("The solution's pH is [round(reagents.ph, 0.01)] and has a temperature of [reagents.chem_temp]K.")
 				else //Otherwise, just show the total volume
 					var/total_volume = 0
-					for(var/datum/reagent/R in reagents.reagent_list)
-						total_volume += R.volume
+					for(var/datum/reagent/current_reagent in reagents.reagent_list)
```
Comment (Watermelon914) [inline code/game/atoms.dm:661]: if these are 100% guaranteed to be reagents, put as anything for them

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

File: code/game/atoms.dm (hunk 13 above)
Comment (Ghilker) [reply code/game/atoms.dm]: i believe those are guaranteed to be reagents, and they should error if there is something that slip in

File: code/game/atoms.dm (hunk 12 above)
Comment (Ghilker) [reply code/game/atoms.dm]: should be guaranteed

File: code/game/atoms.dm (hunk 11 above)
Comment (Ghilker) [reply code/game/atoms.dm]: should be guaranteed

File: code/game/atoms.dm [hunk 14]
```diff
@@ -371,33 +371,31 @@
...
-	if(is_reserved_level(T.z))
-		for(var/A in SSshuttle.mobile)
-			var/obj/docking_port/mobile/M = A
-			if(M.launch_status == ENDGAME_TRANSIT)
-				for(var/place in M.shuttle_areas)
-					var/area/shuttle/shuttle_area = place
-					if(T in shuttle_area)
-						return TRUE
+	if(is_reserved_level(current_turf.z))
+		for(var/obj/docking_port/mobile/mobile_docking_port in SSshuttle.mobile)
```
Comment (Watermelon914) [inline code/game/atoms.dm:379]:
```suggestion
		for(var/obj/docking_port/mobile/mobile_docking_port as anything in SSshuttle.mobile)
```

File: code/game/atoms.dm [hunk 15]
```diff
@@ -1948,40 +1943,40 @@
...
+	if(turf_area.has_gravity) // Areas which always has gravity
+		return turf_area.has_gravity
 	else
 		// There's a gravity generator on our z level
-		if(GLOB.gravity_generators["[T.z]"])
+		if(GLOB.gravity_generators["[gravity_turf.z]"])
 			var/max_grav = 0
-			for(var/obj/machinery/gravity_generator/main/G in GLOB.gravity_generators["[T.z]"])
-				max_grav = max(G.setting,max_grav)
+			for(var/obj/machinery/gravity_generator/main/main_grav_gen as anything in GLOB.gravity_generators["[gravity_turf.z]"])
```
Comment (Watermelon914) [inline code/game/atoms.dm:1976]: if this has to be a gravgen, add `as anything`

**Comment by Watermelon914** [comment]
make sure you test this pr

**Pull request merged**